installer/functions

```diff
@@ -220,6 +224,46 @@ echo_tty() {
     fi
 }
 
+# Outputs colored text to stderr
```

Comment:
Or we could just output to stdout and let the caller redirect... That's more consistent with how `echo` works.